node/src/command.rs

```diff
@@ -96,6 +96,8 @@ fn load_spec(id: &str) -> std::result::Result<Box<dyn sc_service::ChainSpec>, St
         "local" => Box::new(chain_spec::cs_main::local_testnet_config()),
         "staking-dev" => Box::new(chain_spec::cs_staking::development_config()),
         "staking-local" => Box::new(chain_spec::cs_staking::local_staking_config()),
+        "staking-tnet" => Box::new(chain_spec::cs_staking::tnet_staking_config()),
+        "staking-tnet-templ" => Box::new(chain_spec::cs_staking::tnet_templ_staking_config()),
```

Comment:
Should be "staking" and "staking-local"

Reply:
Seconding this suggestion; "staking" and "staking-local" describes the functionality without ambiguity.

it might also help to name the JSON source file to match, not required though. (`staking-local.json`?)

Reply:
Looks like I am messing up with names, "staking-local" is already existing 
one with below functionality. May be for "staking-tnet" can I replace it with "staking"
& "staking-tnet-templ" with "staking-templ" ?

| Chain-Spec | Status | Discp |
| --- | --- | --- |
| "staking-dev" | Existing One | One node development setup, with "Alice" as authority |
| "staking-local" |  Existing One | Two node local setup with "Alice" & "Bob" as authority |
| "staking-tnet" | New One | Multi-node test network with user private keys as authority, & load's chainspec from "node/res/tnet-staking.json" |
| "staking-tnet-templ" | New One | Same as "staking-tnet", but used to generate the "node/res/tnet-staking.json" spec file. |

Reply:
Replaced "staking-tnet" to "staking" & removed "staking-tnet-templ"

| Chain-Spec | Status | Discp |
| --- | --- | --- |
| "staking-dev" | Existing One | One node development setup, with "Alice" as authority |
| "staking-local" |  Existing One | Two node local setup with "Alice" & "Bob" as authority |
| "staking" | New One | Multi-node test network with user private keys as authority, & load's chainspec from "node/res/tnet-staking.json" |